core/server/option.go

```diff
@@ -38,3 +40,21 @@ func WithSession(secret, name string, options ...session.Option) Option {
 		})
 	}
 }
+
+func WithAssets(embedded fs.FS) Option {
+	manager := assets.NewManager(embedded)
+
+	return func(m *mux) {
+		m.Use(func(h http.Handler) http.Handler {
+			return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+				if vlr, ok := r.Context().Value("valuer").(interface{ Set(string, any) }); ok {
+					vlr.Set("assetPath", manager.PathFor)
+				}
+
+				h.ServeHTTP(w, r)
+			})
+		})
+
+		m.HandleFunc(manager.HandlerPattern(), manager.HandlerFn)
```

Comment:
@tatang26 we should ensure this is added after the rest of the things. How do you make sure of that?

Reply:
@paganotoni This is not ensuring that the assets are mapped after of the rest of routes. For this we'll need to define a way the server can hold all mapped routes to append the assets one

Reply:
Well, thats an essential part of this.